### PR TITLE
Added shortcut to collapse sidebar

### PR DESCRIPTION
**Pull request by oleklukasiewicz**
Items resolved / related issues by this PR.
- Closes 
  - #7998 

Add details of changes here.
- Added Ability to Hide/Show Sidebar Visiblity
  - Added shortcut (Ctrl + Alt + S) to toggle sidebar
  - Added custom property IsSidebarVisible in SidebarControl 
  - Added property IsSidebarVisible in SidebarViewModel

How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

**Review by Hk92a — DISMISSED** [review]

**Comment by yaichenbaum** [comment]
@aleksanderlukasiewicz thank you for the pull request. Can you revert the changes that completely hide the sidebar? We haven't approved the request yet since we're still waiting for user feedback.

**Comment by oleklukasiewicz** [comment]
@yair100 Do you mean that if the user hides the sidebar, is it possible to display it again? 

If so, just use Crtl + Alt + S (command toggles sidebar visibility from visible to hidden and vice versa) or restart Files (sidebar is always visible at startup). 

I am also thinking of adding an option to the context menu in the toolbar

**Comment by yaichenbaum** [comment]
@aleksanderlukasiewicz at the moment, the only change that we'd like to introduce is the option to collapse the sidebar using the ctrl + alt + s shortuct. The option to hide the sidebar hasn't been discussed yet and we might not want to implement it.

**Comment by oleklukasiewicz** [comment]
@yair100 ok

**Comment by oleklukasiewicz** [comment]
@yair100 Done!

File: src/Files/UserControls/SidebarControl.xaml.cs [hunk 1]
```diff
@@ -930,6 +930,8 @@ private void Border_KeyDown(object sender, KeyRoutedEventArgs e)
 
         private double originalSize = 0;
 
+        private bool isOnlyIconMode = false;
```
Comment (yaichenbaum) [inline src/Files/UserControls/SidebarControl.xaml.cs:933]: Is this boolean doing anything?

Comment (aleksanderlukasiewicz) [reply src/Files/UserControls/SidebarControl.xaml.cs:933]: Sorry, i forgot about this.

**Comment by yaichenbaum** [comment]
@aleksanderlukasiewicz thank you! Can you add this shortcut to the documentation on the website?

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**